internal/cli/atlas/backup/compliancepolicy/compliancepolicy.go

```diff
@@ -16,6 +16,7 @@ package compliancepolicy
 
 import (
 	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	policies "github.com/mongodb/mongodb-atlas-cli/internal/cli/atlas/backup/compliancepolicy/policies"
```

Comment:
```suggestion
	"github.com/mongodb/mongodb-atlas-cli/internal/cli/atlas/backup/compliancepolicy/policies"
```
you shouldn't need an alias here sine is the same as the pkg name